### PR TITLE
sKt2

### PR DESCRIPTION
**Pull request by s-luresz**

### CODE REVIEW

**Comment by s-luresz** [comment]
Replace the data tables with the unified ones from HIS and FIS (institutions and colors)

File: src/process_FIS_data.py [hunk 1]
```diff
@@ -238,7 +238,13 @@ def write_publications(person_ID: str) -> None:
 
 # Create a tab separated file containing for each institution its short name and 
 # its abbreviation as saved in the dictionary
-file = open(os.path.join(DATA_DIR, "FIS/FIS.inst.abbrev.tbl"), 'w')
+
+# open unified institution abbrevation table Lukasz
+# file = open('../data/FIS/FIS.inst.abbrev.tbl', 'w')
+# was in the conflict 
+# file = open(os.path.join(DATA_DIR, "FIS/FIS.inst.abbrev.tbl"), 'w')
+file = open('../data/FIS/HIS_inst_abbrev.tbl', 'w')
```
[the remark this answers is not in the document]
Comment (MarioStanke) [reply src/process_FIS_data.py:246]: You should have resolved the conflict: understand the intention of the other committer and combine the new variable with your new file name.
The FIS/HIS file name is confusing. The merged file should be specific to neither and need not have a data source name.

**Review by MarioStanke — CHANGES_REQUESTED** [review]
see individual comments

File: data/FIS_inst_abbrev.tbl
[the remark this answers is not in the document]
Comment (s-luresz) [reply data/FIS_inst_abbrev.tbl:1]: This file should be the table of long and short name of institutions of FIS and then it should be unified into one file with HIS. Although it was alredy done in the HIS_inst_abbrev.tbl file.

File: data/HIS_inst_abbrev.tbl
[the remark this answers is not in the document]
Comment (s-luresz) [reply data/HIS_inst_abbrev.tbl:1]: It is changed into `unified_inst_abbrev.tbl` now.

File: src/process_FIS_data.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (s-luresz) [reply src/process_FIS_data.py]: There is used `DATA_DIR` in the code now.

**Review by MarioStanke — APPROVED** [review]
LGTM

**Pull request merged**